docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,65 @@
+    - if the program reaches this line without going to the rule logic, the project is not applicable for this SWH heating distribution system: `NOT_APPLICABLE`
+
+    ## Rule Logic: 
+  - create a boolean to keep track of whether everything matches: `all_match = TRUE`
+  - create an error string: `error_str = ""`
+  - create the compare context string: `compare_context_str = "AppG 11-1 P_RMD Equals U_RMD"`
+  - get a dictionary of all SWH equipment in the proposed model (pumps, distribution, tanks, SWH use, etc): `p_shw_equipment_dict = p_swh_system_and_equip_dict[distribution_id]`
+  - get a dictionary of all SWH equipment in the user model (pumps, distribution, tanks, SWH use, etc): `u_shw_equipment_dict = u_swh_system_and_equip_dict[distribution_id]`
+  - check if the u_shw_equipment exists: `if(u_swh_equipment_dict):`
+    - Compare the distribution in the proposed and user models.  This check relies on the understanding that the RCT team has a method for comparing all elements within an object match (compare_context_pair): `if !compare_context_pair(distribution_id, distribution_id, $, extra_schema_for_SWH_comparison.json, true, compare_context_str, error_str): all_match = FALSE`
+  - otherwise, the distribution equipment doesn't exist in the user model, set all_match to false: `all_match = FALSE; error_str += distribution_id + " not found in the user model"`
+  - continue if all_match is still true: `if all_match`
+    - create a list of all of the SWH equipment types that exist in both the proposed and user models: `shw_equipment_types = set(p_shw_equipment_dict.keys()) | set(u_shw_equipment_dict.keys())`
+    - loop through each of the equipment types: `for shw_equipment_type in shw_equipment_types:`
+      - continue if all_match is still true: `if all_match:`
+        - check if this equipment type matches between the proposed and user models.  First check whether there are the same number equipment: `if len(p_shw_equipment_dict[shw_equipment_type]) == len(u_shw_equipment_dict[shw_equipment_type]):`
+            - look at each equipment type system in the proposed model and see if there is one that is the same in the user model.  This check relies on the understanding that the RCT team has a method for comparing all elements within an object match (compare_context_pair): `for p_SWH_equip_id in p_shw_equipment_dict[shw_equipment_type]:`
+                - if this system is in U_RMD.service_water_heating_distribution_systems, compare the systems: `if p_SWH_equip_id in u_shw_equipment_dict[shw_equipment_type]:`
+                    - use compare_context_pair to compare systems and set all_match to FALSE if the systems don't compare: `if !compare_context_pair(p_SWH_equip_id, p_SWH_equip_id, $, extra_schema_for_SWH_comparison.json, true, compare_context_str, error_str): all_match = FALSE`
```

Comment:
I do not know where to see this function but trust that this works. Maybe Weili should check out this line.

Reply:
Agreed, would like Weili to confirm that the recursive nature of the compare_context_pair function has been communicated properly in this RDS